src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
+document.body.style.position = 'relative';
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const notificationTitle = document.createElement('h2');
+  const notificationMessage = document.createElement('p');
+
+  notification.classList.add('notification', type);
+  notificationTitle.textContent = title;
+  notificationTitle.className = 'title';
+  notificationTitle.style.whiteSpace = 'nowrap';
+  notificationMessage.innerHTML = description;
```

Comment:
Using `innerHTML` to set the description can introduce security risks like XSS (Cross-Site Scripting) if the input is not properly sanitized. If you don't need to render HTML tags in the description, consider using `textContent` instead to avoid these risks.